### ISSUE TITLE
(feat) add a command to query the base denom and denom trace from an IBC denom

### ISSUE THREAD

**Issue by jtieri** [cosmos/relayer#621]
Being able to query the base denom of a token from the IBC denom is a helpful utility function for testing and for operators. 

So given the IBC Denom: 
`ibc/9BBA9A1C257E971E38C1422780CE6F0B0686F0A3085E2D61118D904BFE0F5F5E`

Running this command would return the results below:
`rly q denom-trace osmosis-1 9BBA9A1C257E971E38C1422780CE6F0B0686F0A3085E2D61118D904BFE0F5F5E`

```
denom_trace:
  base_denom: usomm
  path: transfer/channel-165
```

**Comment by mikhailbuslaev** [comment]
https://github.com/cosmos/relayer/pull/625

**Comment by jtieri** [comment]
Closed in #625
